airbyte-tests/src/acceptanceTests/java/io/airbyte/test/acceptance/AcceptanceTests.java

```diff
@@ -157,6 +157,7 @@
  */
 @SuppressWarnings({"rawtypes", "ConstantConditions"})
 @TestMethodOrder(MethodOrderer.OrderAnnotation.class)
+@Disabled
```

Comment:
Is this intended to be added here? If so, why?

Reply:
nope. I was testing how to speed this up. thanks.